redis/src/test/java/org/hibernate/ogm/datastore/redis/test/associations/MapMappingTest.java

```diff
@@ -32,6 +34,7 @@
  * @author Gunnar Morling
  * @author Mark Paluch
  */
+@SkipByGridDialect(GridDialectType.REDIS_HASH)
```

Comment:
Because embedded maps are not supported like it works with JSON. Agree to externalize maps with a key of `Table:id:collectionRole`.